assets/src/components/v2/widget_tree_error_boundary.tsx

```diff
@@ -163,4 +164,18 @@ const WrappedWithLastFetch: React.ComponentType<Omit<Props, "lastFetch">> = (
   return <WidgetTreeErrorBoundary {...props} lastFetch={lastFetch} />;
 };
 
+function withRouter<ComponentProps>(
```

Comment:
Maybe we could add a link [here](https://reactrouter.com/en/main/start/faq#what-happened-to-withrouter-i-need-it). Then we know if this approach is ever redone to not use class components (...de-class-ified? sorry), we shouldn't need this anymore.

Reply:
Yeah good idea.